lib/debuglet.js

```diff
@@ -142,8 +142,8 @@ Debuglet.prototype.scheduleRegistration_ = function(seconds) {
 
   function onError(err) {
     that.logger_.error('Failed to re-register debuggee: ' + err);
-    that.logger_.error('Disabling gcloud debuglet');
-    that.stop(); // fatal error
+    that.scheduleRegistration_(Math.min((seconds + 1) * 2,
```

Comment:
I don't think there is a substantial complexity difference so lets keep the backoff.